frequency-chain/frequency
# Development Genesis Config has Mainnet Schemas

# Goal
The goal of this PR is to add the Schemas in at Genesis from Mainnet when on a development chain

Closes #2041

# Discussion
- Adds a tool `tools/genesis-schemas` that downloads the schemas from mainnet and stuffs them into a json file
- GenesisConfig for Schemas Pallet that loads from a json file schemas to start with at genesis

## How to Test
- You can test the pulling yourself with:
    - `cd tools/genesis-data && npm i && npm run schemas` or use `make genesis-schemas`
- Otherwise just run `make start` and see that it has the same schemas as mainnet with the same names.

# Checklist
